test/react-native/scripts/build-ios.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -e
+
+if [ -z "$REACT_NATIVE_VERSION" ]; then
+    echo "The environment variable 'REACT_NATIVE_VERSION' must be set"
+    exit 1
+fi
+
+cp -r test/react-native/features/fixtures/app test/react-native/features/fixtures/$REACT_NATIVE_VERSION/app
```

Comment:
I guess we will have to be careful that everything in `test/react-native/features/fixtures/app` is compatible with all versions of RN that we test with.

Reply:
Yes.  At the moment the JS is fine to run across all versions of RN, but should it occur that this is not the case this may have to be refactored into separate scenario sets.